### PR TITLE
doc(showcase): add a readme to explain how to update the screenshots for visual testing

### PR DESCRIPTION
**Pull request by fpaul-1A**
## Related issues

*- No issue associated -*

### CODE REVIEW

**Review by sdo-1A — DISMISSED** [review]

File: apps/showcase/scripts/update-screenshots/readme.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# How to generate screenshots for the visual testing
```
Comment (kpanot) [inline apps/showcase/scripts/update-screenshots/readme.md:1]: Maybe you can refer it in the `contributing.md` ?

**Review by kpanot — DISMISSED** [review]

File: apps/showcase/scripts/update-screenshots/readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+To minimize discrepancies on the screenshots between different operating systems, we are generating the screenshots from inside a docker.
+The procedure is the following:
+
+1. Prerequisite: you need a docker setup (or podman), `docker` should be available globally
+2. Build your application with your latest modifications `yarn nx build showcase`.
+3. Serve the application on port 8080 `npx http-server ./apps/showcase/dist/browser`\
+     The script will look on all local interfaces for an app running on `http://localhost:8080`
```
Comment (vscaiceanu-1a) [inline apps/showcase/scripts/update-screenshots/readme.md:9]: you might want to add an exception in `.github/markdown-external-links.config.json` for localhost.

File: apps/showcase/scripts/update-screenshots/readme.md (hunk 1 above)
Comment (fpaul-1A) [reply apps/showcase/scripts/update-screenshots/readme.md:1]: done

File: apps/showcase/scripts/update-screenshots/readme.md (hunk 2 above)
Comment (fpaul-1A) [reply apps/showcase/scripts/update-screenshots/readme.md:9]: done

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -72,12 +73,12 @@ To be able to benefit from this feature, you will need to perform the following
...
 1. Create an account on [Nx Cloud App](https://cloud.nx.app/)
 2. Create a Personal Access Token on [profile page](https://cloud.nx.app/profile/tokens).
-3. Follow [Nx Cloud instruction](https://nx.dev/ci/recipes/security/personal-access-tokens) to setup your previously generated PAT.
+3. Follow [Nx Cloud instruction](https://nx.dev/ci/recipes/security/personal-access-tokens) to set up your previously generated PAT.
 
 When building (`yarn build`) the project on the `main` branch (or another `release/*` branch), the remote cache will be downloaded.
 
 > [!NOTE]
-> Only Otter Team members can write Remote Cache on Nx Cloud, logged in user will be able to use Remote Cache in readonly access.
+> Only Otter Team members can write Remote Cache on Nx Cloud, logged-in user will be able to use Remote Cache in readonly access.
```
Comment (sdo-1A) [inline CONTRIBUTING.md:81]:
```suggestion
> Only Otter Team members can write Remote Cache on Nx Cloud, logged-in users will be able to use Remote Cache in readonly mode.
```

**Review by mrednic-1A — APPROVED** [review]

File: apps/showcase/scripts/update-screenshots/readme.md (hunk 2 above)
Comment (kpanot) [reply apps/showcase/scripts/update-screenshots/readme.md:9]: This tool is considering what is between ``` ` ``` as link? Oo

**Review by kpanot — APPROVED** [review]

File: .github/markdown-links.config.json [hunk 4]
```diff
@@ -2,6 +2,9 @@
   "ignorePatterns": [
     {
       "pattern": "^https?:\/\/.*"
+    },
+    {
+      "pattern": "^https?://(127.0.0.1|localhost)[:/].*"
```
Comment (vscaiceanu-1a) [inline .github/markdown-links.config.json:7]: No need to add the pattern here. As you can see, all urls that start with `http` are ignored already.

**Pull request merged**